code/script.js

```diff
@@ -0,0 +1,85 @@
+//DOM SELECTORS
+
+const projectsContainer = document.getElementById('projects')
+const userContainer = document.getElementById('user-container')
+const userInfo = document.getElementById('user-info')
+const userImg = document.getElementById('user-img')
+
+//APIS TO FETCH FROM
+const username = 'Thereese'
+let reponame
+const API_URL = `https://api.github.com/users/${username}/repos`
+const API_URL_PR = `https://api.github.com/repos/Technigo/${reponame}/pulls`
+
+
+const options = {
+  method: 'GET',
+    headers: {
+      Authorization: 'API_TOKEN'
+    }
+}
+    
+
+//FETCH USERNAME AND PROFILEPIC 
+const getUser = () => {
+  fetch (`https://api.github.com/users/${username}`, options)
+    .then((res) => res.json())
+    .then((data) => {
+      userImg.innerHTML=
+      `<img class="user-image" src=${data.avatar_url}/>`
+      userInfo.innerHTML=
+      `<h1>${data.name}</h1>
+       <h3> | ${data.login}</h3>`
```

Comment:
Great job with fetching the img and username! 
A tip to make it even more clever is to wrap the `<h3>` in `<a>` tag and add `${data.html_url}` to link the username to your github profile.